subworkflows/local/analyse_transcripts.nf

```diff
@@ -56,9 +57,23 @@ workflow ANALYSE_TRANSCRIPTS {
             drop_group_samples_as
         )
 
-        // Generates  config file and runs Aberrant expression module
-        ch_bai_files = ch_bam_ds_bai.collect{ it[2] }.toList()
-        ch_bam_bai_files = ch_bam_files.toList().combine(ch_bai_files)
+
+        // Generates config file and runs Aberrant expression module
+
+        // Sort bam and bai files for stable order
+        // ch_bam_files_sorted = ch_bam_files.toList().map { list ->
+        //     list.sort { a, b -> a.getName() <=> b.getName() }
+        // }
+        // ch_bai_files_sorted = ch_bam_ds_bai.collect{ it[2] }.toList().map { list ->
+        //     list.sort { a, b -> a.getName() <=> b.getName() }
+        // }
+        // ch_bam_bai_files = ch_bam_files_sorted.combine(ch_bai_files_sorted)
```

Comment:
Should we remove this?

Reply:
Yes! I left it there with the intent to remove it after testing / verifying that the change works as intended. If you come to the point of test running / wrapping this before me - just go ahead and remove it.